### PR TITLE
Allow Korean Nickname Use

### PR DESCRIPTION
**Pull request by swan201**
Allow Korean Nickname Using...

### CODE REVIEW

**Comment by ocelotpotpie** [comment]
Had someone who speaks Korean review this. Looks good to me.

Not that my opinion matters much on this project. :)

**Comment by swan201** [comment]
@ocelotpotpie Since Essentials 2.11.1 Version. Suddenly Essentials doesn't Support Korean Nick Name Use, Allow Only Alphabet. Some User Change Code and Upload There Blog. But now. very nice open source share system github is open. so no more need to change code update version 

I really want to merge this simple update code :)

**Comment by vemacs** [comment]
Makes sense to have general Unicode nick support IMO. I don't personally see why nicknames should be limited to valid MC names. I'm thinking of forgoing this and just implementing a max character limit instead.

**Comment by ChanceSD** [comment]
Yes please do that.

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,6 @@ notifications:
     on_failure: always
   email:
     recipients:
-      - "khobbits@ess3.net"
+      - "github@protocodyne.com"
     on_success: change
```
Comment (drtshock) [inline .travis.yml:17]: lol?

**Pull request closed without merging**